README.rst

```diff
@@ -0,0 +1,307 @@
+aws-cli
+=======
+
+
+.. image:: https://travis-ci.org/aws/aws-cli.svg?branch=develop
+   :target: https://travis-ci.org/aws/aws-cli
+   :alt: Build Status
+
+.. image:: https://badges.gitter.im/aws/aws-cli.svg
+   :target: https://gitter.im/aws/aws-cli
+   :alt: Gitter
+
+
+This package provides a unified command line interface to Amazon Web
+Services.
+
+Jump to:
+
+-  `Getting Started <#getting-started>`__
+-  `Getting Help <#getting-help>`__
+-  `More Resources <#more-resources>`__
+
+Getting Started
+---------------
+
+This README is for the AWS CLI version 1. If you are looking for
+information about the AWS CLI version 2, please visit the `v2
+branch <https://github.com/aws/aws-cli/tree/v2>`__.
+
+Requirements
+~~~~~~~~~~~~
+
+The aws-cli package works on Python versions:
+
+-  2.7.x and greater
+-  3.4.x and greater
+-  3.5.x and greater
+-  3.6.x and greater
+-  3.7.x and greater
+-  3.8.x and greater
+
+On 10/09/2019 support for Python 2.6 and Python 3.3 was deprecated and
+support was dropped on 01/10/2020. To avoid disruption, customers using
+the AWS CLI on Python 2.6 or 3.3 will need to upgrade their version of
+Python or pin the version of the AWS CLI in use prior to 01/10/2020. For
+more information, see this `blog
+post <https://aws.amazon.com/blogs/developer/deprecation-of-python-2-6-and-python-3-3-in-botocore-boto3-and-the-aws-cli/>`__.
+
+*Attention!*
+
+*We recommend that all customers regularly monitor the*\ `Amazon Web
+Services Security Bulletins
+website <https://aws.amazon.com/security/security-bulletins>`__\ *for
+any important security bulletins related to aws-cli.*
+
+Installation
+~~~~~~~~~~~~
+
+The safest way to install the AWS CLI is to use
+`pip <https://pip.pypa.io/en/stable/>`__ in a ``virtualenv``:
+
+::
+
+   $ python -m pip install awscli
+
+or, if you are not installing in a ``virtualenv``, to install globally:
+
+::
+
+   $ sudo python -m pip install awscli
+
+or for your user:
+
+::
+
+   $ python -m pip install --user awscli
+
+If you have the aws-cli package installed and want to upgrade to the
+latest version you can run:
+
+::
+
+   $ python -m pip install --upgrade awscli
+
+This will install the aws-cli package as well as all dependencies.
+
+*Note*
```

Comment:
It might be worth try the Note directive for RST: https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html. Looking at the rendering it looks like it does not make `six` and `disutils` code literals.